.circleci/config.yml

```diff
@@ -1,16 +1,15 @@
 version: 2
 jobs:
   build:
-    working_directory: /usr/src/app
```

Comment:
why remove this line?

eg. https://github.com/nuxt/nuxt.js/blob/dev/.circleci/config.yml#L4